lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -12,5 +12,41 @@ def fulfillments(options = {})
       fo_fulfillments = get(:fulfillments, options)
       fo_fulfillments.map { |fof| FulfillmentOrderFulfillment.new(fof.as_json) }
     end
+
+    def move(new_location_id:)
+      body = { fulfillment_order: { new_location_id: new_location_id } }.to_json
+      keyed_fos = load_keyed_attributes_from_response(post(:move, {}, body))
+      if keyed_fos&.fetch('original_fulfillment_order', nil)&.attributes
+        load(keyed_fos['original_fulfillment_order'].attributes, false, true)
+      end
+      keyed_fos
+    end
+
+    def cancel
+      keyed_fos = load_keyed_attributes_from_response(post(:cancel, {}, only_id))
+      if keyed_fos&.fetch('fulfillment_order', nil)&.attributes
+        load(keyed_fos['fulfillment_order'].attributes, false, true)
+      end
+      keyed_fos
+    end
+
+    def close
+      load_attributes_from_response(post(:close, {}, only_id))
```

Comment:
Is it safe to assume that this will just return the `fulfillment order response` received from Shopify/Shopify without doing any parsing (aka doing `FulfillmentOrder.new(fo_attributes)` )?

Reply:
We don't need to do `FulfillmentOrder.new(fo_attributes)` because the subject of the `.close` method is itself an instance of `FulfillmentOrder` and just updating its attributes follows the ActiveResource way of making network operations behave like ActiveRecord mutating methods. It's a bit weird but I'm learning to follow it.